src/pkg/packaging-tools/framework.sharedfx.targets

```diff
@@ -221,16 +184,45 @@
       <TrimPkgsFromDeps Include="microsoft.netcore.dotnethostresolver" />
     </ItemGroup>
 
-    <ProcessSharedFrameworkDeps AssetsFilePath="$(SharedFrameworkAssetsFile)"
+    <ProcessSharedFrameworkDeps AssetsFilePath="$(ProjectAssetsFile)"
                                 DepsFilePath="$(SharedFrameworkDepsFile)"
                                 PackagesToRemove="@(TrimPkgsFromDeps)"
                                 Runtime="$(RuntimeGraphGeneratorRuntime)"
-                                BuildToolsTaskDir="$(BuildToolsTaskDir)" />
+                                BuildTasksAssemblyPath="$(PackagingTaskDir)Microsoft.DotNet.Build.Tasks.Packaging.dll" />
   </Target>
 
   <Target Name="CleanPackages" BeforeTargets="Clean">
     <RemoveDir Directories="$(PublishDir);$(RestorePackagesPath)" />
     <Delete Files="$(ProjectAssetsFile)" />
   </Target>
 
+  <!--
+    Ensure the pkgproj is already built. This isn't a real ProjectReference because the SDK doesn't
+    seem to handle pkgproj nicely for Publish. For now we're using its nupkg directly.
```

Comment:
This comment implies to me additional changes will follow?  Is that going to be part of the arcade migration?

Reply:
More to come, but only as part of the move to share shared framework generation tooling in the Arcade SDK. Adding a note.